### DISCUSSION TITLE
make PythonAPI error [clang++: error: clang frontend command failed due to signal]

### DISCUSSION THREAD

**Discussion by mohussein97** [carla-simulator/carla#7610] [Q&A]
I am using ubuntu 20.04, python 3.9,carla: 0.9.15, unreal engine: carla 4.26 I have 96 GB Ram processor i9-14900k and rtx 4080 master and have followed the official documentation exactly. yet when I run make PythonAPI I got that error  and it crashes [clang++: error: clang frontend command failed due to signal (use -v to see invocation)
clang version 10.0.1 
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /home/mo/UnrealEngine_4.26/Engine/Extras/ThirdPartyNotUE/SDKs/HostLinux/Linux_x64/v17_clang-10.0.1-centos7/x86_64-unknown-linux-gnu/bin
clang++: note: diagnostic msg: PLEASE submit a bug report to https://bugs.llvm.org/ and include the crash backtrace, preprocessed source, and associated run script.
clang++: note: diagnostic msg: 
********************

PLEASE ATTACH THE FOLLOWING FILES TO THE BUG REPORT:
Preprocessed source(s) and associated run script(s) are located at:
clang++: note: diagnostic msg: /tmp/DOMPrint-015fd1.cpp
clang++: note: diagnostic msg: /tmp/DOMPrint-015fd1.sh
clang++: note: diagnostic msg: 

********************

[346/378] Building CXX object tests/CM....dir/src/XSValueTest/XSValueTest.cpp.o
ninja: build stopped: subcommand failed.
] ![make PythonAPI error](https://github.com/carla-simulator/carla/assets/168490592/34aca340-0582-4acf-9f64-a4173ab397f9)

**Comment by MOHAMMEDHUSSEIN94** [comment]
try clang-8 instead of clang-10

**Reply by mohussein97** [reply]
clang-8 refuses to be installed on ubuntu 20
